angular-ngrx-scss/src/app/shared/components/repo-controls/repo-controls.component.html

```diff
@@ -52,7 +53,7 @@
           {{ (selectSortFilter$ | async)?.split('_')?.join(' ') | lowercase }}
         </span>
       </p>
-      <span class="results__clear-container">
+      <span class="results__clear-container" (click)="handleClearClick()">
```

Comment:
Could we change `span` to `button type="button"`?

Reply:
changed